From f5e24a76671ce566ecbff2de21ce997a71f1a834 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cl=C3=A9ment=20Le=20Goffic?= <clement.legoffic@foss.st.com>
Date: Wed, 30 Apr 2025 14:05:55 +0200
Subject: [PATCH 1682/1695] dts: stm32: add I2C3 SCL rising and falling time
 property
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add 'i2c-scl-[rising|falling]-time-ns' properties values on
stm32mp215f-dk.dts.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8815dbb3474860c3333ac4d6e8f57580d82bdb4a dts: stm32: add I2C3 SCL rising and falling time property]
Change-Id: Ida26c8727599ecab9bba7db36869ca65610bb4d2
Signed-off-by: Clément Le Goffic <clement.legoffic@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/455170
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index afb130307..fa80eb782 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -59,8 +59,8 @@
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&i2c3_pins_a>;
 	pinctrl-1 = <&i2c3_sleep_pins_a>;
-	i2c-scl-rising-time-ns = <185>;
-	i2c-scl-falling-time-ns = <20>;
+	i2c-scl-rising-time-ns = <31>;
+	i2c-scl-falling-time-ns = <2>;
 	clock-frequency = <400000>;
 	status = "okay";
 	#address-cells = <1>;
-- 
2.47.2

